home.html

```diff
@@ -1,33 +0,0 @@
-<html>
-<head>
-<link rel="stylesheet" type="text/css" href="theme.css">
-<script language="javascript" type="text/javascript" src="myscripts.js">
```

Comment:
Use a good file name describing content of the js file. This is a good practice when you write production code where you have so many js file included on html file.